test/behavior/tautological_int_comparison.zig

```diff
@@ -0,0 +1,108 @@
+const std = @import("std");
+const builtin = @import("builtin");
+
+// All conditional expressions in this test have a guaranteed
+// result, so one branch should never be analyzed.
+// https://github.com/ziglang/zig/issues/2146
```

Comment:
Is this the remaining TODO for stage1?

Reply:
Not a TODO, just a link to the original proposal. I'm not sure why I left that in there, so bye bye comment.